src/pretix/base/models/vouchers.py

```diff
@@ -296,6 +296,14 @@ class Voucher(LoggedModel):
         verbose_name=_("Shows hidden products that match this voucher"),
         default=True
     )
+    all_addons_included = models.BooleanField(
+        verbose_name=_("Offer all add-on products for free when redeeming this voucher"),
+        default=False
+    )
+    all_bundles_included = models.BooleanField(
+        verbose_name=_("Include all bundled products without a designated price when redeeming this voucher"),
```

Comment:
That help-text is a bit ambigious – it could mean, only bundled products without a designated price are included, when this voucher is redeemed. Does not really make sense if one thinks about it, but IMHO the following is easier to understand:

```suggestion
        verbose_name=_("Include all bundled products for free when redeeming this voucher"),
```